### ISSUE TITLE
Problem with WPF data templates/user controls/tab controls

### ISSUE THREAD

**Issue by objorke** [oxyplot/oxyplot#175]
The PlotView control does not behave correctly with data templates, user controls and tab controls.
See `Source\Examples\WPF\WpfExamples\Examples\UserControlDemo`

**Comment by UberError** [comment]
Just FYI for anyone viewing this issue. My workaround was to save the generated images with:
```
PngExporter.Export(PlotModel, path, 90, 30, OxyColors.Transparent);
```
Where path is a subdirectory in:
```
Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData)
```

**Comment by draagyn** [comment]
I believe this was fixed by #423

**Comment by tibel** [comment]
@draagyn Yes this is fixed. Thanks.
